pb/lmsysmd/dataset/v1/dataset.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package lmsysmd.dataset.v1;
+
+import "buf/validate/validate.proto";
+import "google/protobuf/timestamp.proto";
+
+service DatasetService {
+  rpc CreateDataset(CreateDatasetRequest) returns (CreateDatasetResponse) {
+    option idempotency_level = IDEMPOTENT;
+  }
+}
+
+message CreateDatasetRequest {
+  string url = 1 [ 
+    (buf.validate.field).required = true,
+    (buf.validate.field).string.uri = true 
+    ];
+}
+
+message CreateDatasetResponse {
+  repeated Dataset dataset = 1;
+}
+
+// Corresponds to one line on https://docs.google.com/spreadsheets/d/1-skT0JRJREJfn5qIQtSwyz2KVHMfLPJy0hOBfjyggZA/edit?gid=0#gid=0
+message Dataset {
```

Comment:
do we also want to include a uint32 dataset_id?

Reply:
I was planning on just using the autoincrement id from postgres

Reply:
so let's separate the database concerns from the api for now - as a consumer of the api, you would want to get the id so you could pass it to other apis

Reply:
true true